### ISSUE TITLE
Approving social media: embed latest youtube videos from channel for easier approval

### ISSUE THREAD

**Issue by fangeugene** [the-blue-alliance/the-blue-alliance#1551]
See https://github.com/the-blue-alliance/the-blue-alliance/pull/1389#issuecomment-225730626

Edit: Make sure different channel url patterns are supported properly.

**Comment by phil-lopreiato** [comment]
Nice find @technonerdz! 

> Edit: Make sure different channel url patterns are supported properly.

I think at this point, we should have already dealt with all the channel urls and just have the foreign key, and you can render the full url the way the current review page does

**Comment by fangeugene** [comment]
> I think at this point, we should have already dealt with all the channel urls and just have the foreign key, and you can render the full url the way the current review page does

Yeah I don't know if @technonerdz's ytchannelembed.com script properly handles the foreign key redirections.

**Comment by technonerdz** [comment]
When I tried with my team, I was not able to do it with my channel custom name: https://www.youtube.com/c/evolution2626first I was only able with this: https://www.youtube.com/channel/UCLm-jhnXUfkqV_NTTMgvasg

**Comment by technonerdz** [comment]
I don't exactly know what you guys mean by Foreign Key but if it is the key behind youtube.com/channel/ then it is ok.

If a foreign key looks like that: UCLm-jhnXUfkqV_NTTMgvasg then it will definitely work!

**Comment by fangeugene** [comment]
The foreign key we use is human readable. So like "officialfirst" in
https://www.youtube.com/officialfirst

**Comment by phil-lopreiato** [comment]
So YouTube URLs have an irritating number of cases. We currently handle [three different url patterns](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/helpers/media_helper.py#L54), all of which deal with the human readable channel name. 

We support `youtube.com/user/blah` and `youtube.com/blah` for regular accounts and `youtube.com/c/blah` for Google Apps accounts. We don't want to support the internal channel IDs, because it makes the team pages that show the profile difficult to read - we want to avoid long machine generated strings there.

**Comment by technonerdz** [comment]
I think that it will work just fine.
